src/components/MediaPlayerCarousel.tsx

```diff
@@ -200,19 +203,22 @@ export class MediaPlayerCarousel extends React.PureComponent<Props, State> {
               onLayout={(event) => {
                 this.setState({ explosionOrigin: event.nativeEvent.layout.y })
               }}
-              disabled={boostPaymentLoading}
+              disabled={boostWasSent}
```

Comment:
Should this be the `boostIsSending` property instead? To disable the button while the boost is being sent?

Reply:
Ah, it should be both of the boost state properties I think, so it waits until the button restores the usual Boost text.